src/client/order-components/GoogleAPI.js

```diff
@@ -2,7 +2,12 @@
 let map;
 const formatAnswer = (mode, value) => {
   if (!value) {
-    return 'not available';
+    // return 'not available';
+    if (mode === 'fast') {
+      return '5 minutes';
```

Comment:
how about `No matching rides available`, though i know thats a bit wordy. N/A looks like an error to me